Gemfile

```diff
@@ -1,6 +1,10 @@
 source "https://rubygems.org"
 
-gem "xcode-install", ">= 2.0.10"
+gem "xcode-install", ">= 2.0.10" # needed for running xcode-install related tests
+
+gem "danger", ">= 4.2.1", "< 5.0.0"
+gem "danger-junit", ">= 0.7.3", "< 1.0.0"
+
```

Comment:
Nope, because it's more restrictive. In this example we want **at least** `4.2.1`, however we are also okay with `4.3`, but not with `5.0` (breaking). That's the only way to specify it like this and we use it in [fastlane.gemspec](https://github.com/fastlane/fastlane/blob/master/fastlane.gemspec#L29)